src/config/wmodules-agent-upgrade.c

```diff
@@ -30,6 +35,10 @@ int wm_agent_upgrade_read(xml_node **nodes, wmodule *module) {
         data->agent_config.upgrade_wait_max = 3600;
```

Comment:
Please, create defines for the default values of `upgrade_wait_start`, `upgrade_wait_max` and `upgrade_wait_factor_increase`